LayoutTests/mathml/presentation/scripts-subsup.html

```diff
@@ -1,28 +1,26 @@
 <!doctype html>
-<meta name="fuzzy" content="maxDifference=187-255; totalPixels=213-318" />
+<meta name="fuzzy" content="maxDifference=0-255; totalPixels=0-520" />
```

Comment:
Which such big fuzzy tolerance, I wonder how relevant the test is now... Some suggestions below.